org.lflang/src/org/lflang/generator/ReactionInstanceGraph.java

```diff
@@ -83,6 +85,10 @@ public ReactionInstanceGraph(ReactorInstance main) {
     public void rebuild() {
         this.clear();
         addNodesAndEdges(main);
+
+        // FIXME: Use {@link TargetProperty#EXPORT_DEPENDENCY_GRAPH}.
```

Comment:
FIXME

Reply:
I don't know what this is. I guess I'll just remove the `FIXME`.

Reply:
```suggestion
```